code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -525,7 +525,7 @@ Auto Patrol[]"},
 		return
 	if(iscarbon(A))
 		var/mob/living/carbon/C = A
-		if(C.canmove || arrest_type) // CIT CHANGE - makes sentient ed209s check for canmove rather than !isstun.
+		if(CHECK_MOBILITY(C, MOBILITY_STAND|MOBILITY_MOVE|MOBILITY_USE) || arrest_type) // CIT CHANGE - makes sentient ed209s check for canmove rather than !isstun.
```

Comment:
CHECK_MULTIPLE_BITFLAGS is used elsewhere and I assume there's a reason for this

Reply:
the reason it doesn't do this is because any of those are considered as "the suspect is still able to resist" so i'm making it check any

multiple checks for ALL of them existing, if someone can't stand but still can move/use i stil lwant them to get stunned